installing/installing_ibm_power/installing-ibm-cloud-account-power-vs.adoc

```diff
@@ -0,0 +1,39 @@
+:_content-type: ASSEMBLY
+[id="installing-ibm-cloud-account-power-vs"]
+= Configuring an IBM Cloud account
+include::_attributes/common-attributes.adoc[]
+:context: installing-ibm-cloud-account-power-vs
+
+toc::[]
+
+Before you can install {product-title}, you must configure an IBM Cloud account.
+
+:FeatureName: {ibmpowerProductName} Virtual Server using installer-provisioned infrastructure
+include::snippets/technology-preview.adoc[]
+
+[id="prerequisites_installing-ibm-cloud-account-power-vs"]
+== Prerequisites
+
+* You have an IBM Cloud account with a subscription. You cannot install {product-title} on a free or trial IBM Cloud account.
+
+include::modules/quotas-and-limits-ibm-power-vs.adoc[leveloffset=+1]
+
+[id="configuring-dns-resolution-powervs"]
+== Configuring DNS resolution
+
+How you configure DNS resolution depends on the type of {product-title} cluster you are installing:
+
+* If you are installing a public cluster, you use IBM Cloud Internet Services (CIS).
+* If you are installing a private cluster, you use IBM Cloud DNS Services (DNS Services)
```

Comment:
@mjturek could you confirm if above change is okay to add here ?
It is as per this comment : https://github.com/openshift/openshift-docs/pull/47558/files#r1143851863 fyi.

Reply:
yep looks good. I might've said "you must use" but this works. Thanks!